pkgs/by-name/us/ustr/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ustr";
+  version = "1.0.4";
+
+  src = fetchgit {
+    url = "http://www.and.org/ustr/ustr.git";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-pQrQy+S9fVFl8Mop4QmwEAXGiBSheQE4HgAZ4srFz64=";
+  };
+
+  # Fixes bogus warnings that failed libsemanage
+  patches = [ ./va_args.patch ];
+
+  # Work around gcc5 switch to gnu11
+  env.NIX_CFLAGS_COMPILE = "-std=gnu89";
+
+  # Fix detection of stdint.h
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace "have_stdint_h=0" "have_stdint_h=1"
+    substituteInPlace ustr-import.in \
+      --replace "USTR_CONF_HAVE_STDINT_H=0" "USTR_CONF_HAVE_STDINT_H=1"
```

Comment:
```suggestion
      --replace-fail "USTR_CONF_HAVE_STDINT_H=0" "USTR_CONF_HAVE_STDINT_H=1"
```

Reply:
Oh thanks, it also help fix `USTR_CONF_HAVE_STDINT_H=0` => `USTR_CONF_HAVE_STDINT_H 0`